src/components/common/GitInfoMaterial.tsx

```diff
@@ -50,9 +55,10 @@ export default function GitInfoMaterial({
             setSearchText('')
             setSearchApplied(false)
         } else if (selectedMaterial.searchText !== searchText) {
-            setSearchText(selectedMaterial.searchText)
+            //setSearchText(selectedMaterial.searchText)
```

Comment:
remove commented or uncomment if need to be used

Reply:
done